tracing-core/src/metadata.rs

```diff
@@ -261,6 +261,22 @@ impl Level {
     pub const TRACE: Level = Level(LevelInner::Trace);
 }
 
+impl fmt::Display for Level {
+    fn fmt(&self, f: &mut fmt::Formatter) -> fmt::Result {
+        write!(
```

Comment:
Rather than `write!`, can we use [`Formatter::pad`](https://doc.rust-lang.org/std/fmt/struct.Formatter.html#method.pad) to format the strings? 

Since the different `Level`s are different lengths, there are probably a lot of situations where users might format a `Level` using a format spec that pads it to 5 characters. If we use `Formatter::pad`, the formatter will handle this. 

I think the resulting code would then look like this:
```rust
    fn fmt(&self, f: &mut fmt::Formatter) -> fmt::Result {
        match *self {
            Level::TRACE => f.pad("TRACE"),
            // ...
        }
    }
```
```

Reply:
Makes sense!